doc/classes/AnimatedSprite2D.xml

```diff
@@ -5,34 +5,74 @@
+				This method can be used to determine if the current animation is playing backwards.
+			</description>
+		</method>
+		<method name="is_playing" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if playing an animation.
+			</description>
+		</method>
+		<method name="pause">
+			<return type="void" />
+			<description>
+				Pauses the currently playing animation. The [member frame] will be kept and calling [method play] or [method play_backwards] without arguments will resume the animation.
+				See also [method stop].
+			</description>
+		</method>
 		<method name="play">
 			<return type="void" />
-			<param index="0" name="anim" type="StringName" default="&amp;&quot;&quot;" />
-			<param index="1" name="backwards" type="bool" default="false" />
+			<param index="0" name="name" type="StringName" default="&amp;&quot;&quot;" />
+			<param index="1" name="custom_speed" type="float" default="1.0" />
+			<param index="2" name="from_end" type="bool" default="false" />
+			<description>
+				Plays the animation with key [param name]. If [param custom_speed] is negative and [param from_end] is [code]true[/code], the animation will play backwards (which is equivalent to calling [method play_backwards]).
+				If this method is called with that same animation [param name], or with no [param name] parameter, the assigned animation will resume playing if it was paused.
+			</description>
+		</method>
+		<method name="play_backwards">
+			<return type="void" />
+			<param index="0" name="name" type="StringName" default="&amp;&quot;&quot;" />
+			<description>
+				Plays the animation with key [param name] in reverse.
+				This method is a shorthand for [method play] with [code]custom_speed = -1.0[/code] and [code]from_end = true[/code], so see its description for more information.
+			</description>
+		</method>
+		<method name="set_frame_advanced">
+			<return type="void" />
+			<param index="0" name="frame" type="int" />
+			<param index="1" name="is_backward" type="bool" default="false" />
+			<param index="2" name="reset" type="bool" default="true" />
 			<description>
-				Plays the animation named [param anim]. If no [param anim] is provided, the current animation is played. If [param backwards] is [code]true[/code], the animation is played in reverse.
-				[b]Note:[/b] If [member speed_scale] is negative, the animation direction specified by [param backwards] will be inverted.
+				The setter of [member frame] resets the [member frame_progress], but this method allows more detailed settings.
+				If [param is_backward] is [code]true[/code], the [member frame_progress] is reset to [code]1.0[/code].
+				If [param reset] is [code]false[/code], the [member frame_progress] is not reset.
```

Comment:
I don't understand why this method is needed. Maybe `set_frame_advanced(frame: int, frame_progress: float = 0.0, backwards: bool = false)`? Or use `float` (`5.2` is frame `5`, progress `0.2`)?

Reply:
This exposes a method that is frequently called internally: set_frame is exposed to gdscript as a wrapper function for this function, since it registers as a property and cannot have any other arguments.

But well, it may be fine to expose such as set_frame_without_reset(), which does not reset progress as a separate method. I'll think about it later.

Reply:
Replaced by `set_frame_and_progress(int p_frame, real_t p_progress)`